src/main/java/org/jvnet/hudson/plugins/thinbackup/ThinBackupPluginImpl.java

```diff
@@ -61,17 +58,21 @@ public class ThinBackupPluginImpl extends Plugin {
     private boolean failFast = true;
 
     @Override
-    public void start() throws Exception {
-        super.start();
+    public boolean configure(StaplerRequest request, JSONObject jsonObject) {
+        return true;
+    }
```

Comment:
I might be missing something here, but the default implementation in the superclass "calls `StaplerRequest#bindJSON(Object, JSONObject)}`, appropriate when your implementation has getters and setters for all fields." This seems to be the case here, so I'm struggling to understand why we want to override the default implementation. If this is intentional, it might be a good idea to clarify the reasoning in a code comment.

Reply:
you are right, that shouldn't be needed, I think true is anyhow the default. Nik and I played with that but I forgot to clean it up